packages/askui-nodejs/src/utils/proxy/certs/cert_proxy.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Where are all the certificates in `certs` used? Are you sure they should be committed?